src/main/kotlin/FileUtils.kt

```diff
@@ -0,0 +1,108 @@
+import androidx.compose.material3.ColorScheme
+import androidx.compose.runtime.Composable
+import java.io.File
+import java.io.FileNotFoundException
+import java.nio.file.FileAlreadyExistsException
+import javax.swing.JFileChooser
+import javax.swing.filechooser.FileSystemView
+import kotlin.io.path.Path
+import kotlin.io.path.createFile
+
+@Composable
+internal fun FileChooserDialog(
+    title: String,
+    onResult: (result: File) -> Unit,
+    onCancel: () -> Unit
+) {
+    val fileChooser = JFileChooser(FileSystemView.getFileSystemView()).apply {
+        currentDirectory = File(System.getProperty("user.dir"))
+        dialogTitle = title
+        fileSelectionMode = JFileChooser.DIRECTORIES_ONLY
+        isAcceptAllFileFilterUsed = true
+        selectedFile = null
+        currentDirectory = null
+    }
+    if (fileChooser.showOpenDialog(null) == JFileChooser.APPROVE_OPTION) {
+        val file = fileChooser.selectedFile
+        onResult(file)
+    } else {
+        onCancel.invoke()
+    }
+}
+
+internal fun saveColorsToFile(
+    currentColorPalette: ColorScheme,
+    file: File,
+    onResult: (success: Boolean, message: String) -> Unit
+) {
+    try {
+        val path = file.path
+        val outputFile = Path(path, "colors.kt").createFile().toFile()
+        outputFile.apply {
```

Comment:
How does this work when we want to export both light & dark theme? Will it overwrite to an existing `colors.kt`?

Reply:
For now it will just export the colors that are shown, not dark/light colors. Maybe we can expand on this later 🤔 Good idea! I didn't think about that 🤔

Reply:
Sounds good! We can definitely take it up later